kit/Kit.cpp

```diff
@@ -374,42 +365,39 @@ class LinkOrCopy
         }
 
         LOG_INF("linkOrCopy " << linkOrCopyTypeString(type) << " from [" << source << "] to ["
-                              << destination.toString() << "].");
+                              << destination << "].");
 
-        linkOrCopyType = type;
-        sourceForLinkOrCopy = source;
-        if (sourceForLinkOrCopy.back() == '/')
-            sourceForLinkOrCopy.pop_back();
-        destinationForLinkOrCopy = destination;
-        linkOrCopyFileCount = 0;
-        linkOrCopyStartTime = std::chrono::steady_clock::now();
+        Type = type;
+        SourcePath = source;
+        if (SourcePath.back() == '/')
+            SourcePath.pop_back();
+        DestPath = destination;
+        FileCount = 0;
+        StartTime = std::chrono::steady_clock::now();
 
-        if (nftw(source.c_str(), linkOrCopyFunction, 10, FTW_ACTIONRETVAL|FTW_PHYS) == -1)
+        if (nftw(source.c_str(), linkOrCopyFunction, 10, FTW_ACTIONRETVAL | FTW_PHYS) == -1)
         {
             LOG_SYS("linkOrCopy: nftw() failed for '" << source << '\'');
         }
 
-        if (linkOrCopyVerboseLogging)
```

Comment:
The code change -looks- as if we now un-conditionally LOG_INF in this case, add another ::now() call into that code-path etc.